src/pages/RFOX/hooks/useCurrentApyQuery.ts

```diff
@@ -49,7 +49,9 @@ export const useCurrentApyQuery = ({ stakingAssetId }: useCurrentApyQueryProps)
       if (!stakingAsset) return
       if (!totalStakedCryptoCurrencyQuery?.data) return
 
-      const previousEpoch = epochs[0]
+      const previousEpoch = epochs.reduce((latest, current) => {
+        return current.number > latest.number ? current : latest
+      })
```

Comment:
I was about to comment on this:
```ts
const orderedEpochs = epochs.sort((a, b) => b.number - a.number)
const previousEpoch = orderedEpochs[0]
```

But well I'm happy with both, don't want to be so nitpicking